src/explore-education-statistics-admin/src/pages/publication/components/PublicationDetailsForm.tsx

```diff
@@ -80,11 +81,15 @@ const PublicationDetailsForm = ({
           ...(initialValues ?? {
             theme: '',
             title: '',
+            summary: '',
             topicId: '',
           }),
         }}
         validationSchema={Yup.object<PublicationDetailsFormValues>({
           title: Yup.string().required('Enter a title'),
+          summary: Yup.string()
+            .required('Enter a summary')
+            .max(160, 'Summary must be 160 characters or less'),
```

Comment:
Not sure if PublicationDetailsForm is missing tests or if it's tested through some other component?

Reply:
I think the tests are in `PublicationDetailsPage.test.tsx`. I've added a test to check for a publication summary validation error, the same as in `PublicationForm`.